### ISSUE TITLE
[ WARNING ] NaN or Inf found in input tensor.

### ISSUE THREAD

**Issue by Jabari-z** [Lornatang/SRGAN-PyTorch#18]
I got these output，why？please help me！  myGPU ： 2080Ti  driver 11.2
[ WARNING ] You have chosen a specific GPU. This will completely disable data parallelism.
[ INFO ] Use GPU: 0 for training.
[ INFO ] Creating model `srgan_2x2`.
[ INFO ] Losses function information:
        Pixel:       MSELoss
        Content:     VGG19_36th
        Adversarial: BCELoss
[ INFO ] Optimizer information:
        PSNR learning rate:          0.0001
        Discriminator learning rate: 0.0001
        Generator learning rate:     0.0001
        PSNR optimizer:              Adam, [betas=(0.9,0.999)]
        Discriminator optimizer:     Adam, [betas=(0.9,0.999)]
        Generator optimizer:         Adam, [betas=(0.9,0.999)]
        PSNR scheduler:              None
        Discriminator scheduler:     StepLR, [step_size=self.gan_epochs // 2, gamma=0.1]
        Generator scheduler:         StepLR, [step_size=self.gan_epochs // 2, gamma=0.1]
[ INFO ] Load training dataset
[ INFO ] Dataset information:
        Train Path:              /home/zhang/code-space/data/train
        Test Path:               /home/zhang/code-space/data/test
        Number of train samples: 800
        Number of test samples:  100
        Number of train batches: 50
        Number of test batches:  7
        Shuffle of train:        True
        Shuffle of test:         False
        Sampler of train:        False
        Sampler of test:         None
        Workers of train:        4
        Workers of test:         4
[ INFO ] Turn on mixed precision training.
[ INFO ] Train information:
        PSNR-oral epochs: 500
        GAN-oral epochs:  500
Epoch: [0][ 0/50]       Time 4.5959 (4.5959)    Loss 0.405921 (0.405921)
Epoch: [0][ 5/50]       Time 0.0693 (1.0562)    Loss 0.087571 (0.207982)
Epoch: [0][10/50]       Time 0.0969 (0.8460)    Loss 0.068509 (0.141885)
Epoch: [0][15/50]       Time 0.0597 (0.7817)    Loss 0.040221 (0.114032)
Epoch: [0][20/50]       Time 2.5262 (0.8705)    Loss 0.027806 (0.094019)
Epoch: [0][25/50]       Time 0.0585 (0.7959)    Loss 0.032152 (0.081912)
Epoch: [0][30/50]       Time 0.0532 (0.7511)    Loss 0.016601 (0.072890)
Epoch: [0][35/50]       Time 0.0593 (0.7287)    Loss 0.029243 (0.066070)
Epoch: [0][40/50]       Time 3.0444 (0.7835)    Loss 0.024154 (0.060616)
Epoch: [0][45/50]       Time 0.0654 (0.7643)    Loss 0.021094 (0.056146)
PSNR: nan SSIM: nan LPIPS: 0.5387 GMSD: nan: 100%|███████████████████████████████████████████| 7/7 [00:05<00:00,  1.33it/s]
[ WARNING ] NaN or Inf found in input tensor.
[ WARNING ] NaN or Inf found in input tensor.
[ WARNING ] NaN or Inf found in input tensor.
[ WARNING ] NaN or Inf found in input tensor.
Epoch: [1][ 0/50]       Time 3.3172 (3.3172)    Loss nan (nan)
[ WARNING ] NaN or Inf found in input tensor.
[ WARNING ] NaN or Inf found in input tensor.
Epoch: [1][ 5/50]       Time 0.0581 (1.0160)    Loss 0.020317 (nan)
[ WARNING ] NaN or Inf found in input tensor.
[ WARNING ] NaN or Inf found in input tensor.
[ WARNING ] NaN or Inf found in input tensor.
Epoch: [1][10/50]       Time 0.5341 (0.9001)    Loss nan (nan)

**Comment by Lornatang** [comment]
Please check the following data format.

**Comment by Jabari-z** [comment]
I download the dataset using scrip（ /data/download_dataset.sh）and i train success sometime.
When I was debugging yesterday, I found that loss is NaN. 

One more question,I read the data set code, LR img are resized from HR img, that meaning I have the HR img is OK?

**Comment by Lornatang** [comment]
Logically speaking, yes. But it is necessary to ensure that the high-resolution image can be divisible by 4.

**Comment by Lornatang** [comment]
@Jabari-z Your dataset shape ? And your command?

**Comment by Jabari-z** [comment]
DIV2K 800 train 100val , 
python -a srgan_2x2 --upscale-factor=2 --gpu=0 data

**Comment by Lornatang** [comment]
Turn off data enhancement and have a try?

**Comment by Jabari-z** [comment]
YES, I tried. it's OK, it is still running. Thinks. 
BUT, why  data augument effected it.

**Comment by Lornatang** [comment]
Data augment operator likely add double dataset.
